src/node_crypto.cc

```diff
@@ -1795,6 +1795,25 @@ static bool SafeX509ExtPrint(BIO* out, X509_EXTENSION* ext) {
 }
 
 
+static void addFingerprintDigest(unsigned char *md,
```

Comment:
Nit: could you make the `*` [left-leaning](https://github.com/nodejs/node/blob/master/CPP_STYLE_GUIDE.md#left-leaning-c-style-asterisks-for-pointer-declarations)?

Reply:
Ah. Whops :) Thanks!
I've amended the commit